numba_dppy/dppy_devicearray.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import dpctl.memory as dpctl_mem
+import dpctl
+from numba.core import types
+from numba.np import numpy_support
+
+
+class DPPYDeviceArray(object):
```

Comment:
Is is necessary to name it with DPPY prefix? It is from dppy package already.

Reply:
Changed.